src/types/synchronization.d.ts

```diff
@@ -2,11 +2,22 @@ import { MetadataImportResponse, MetadataImportStats } from "./d2";
 import SyncReport from "../models/syncReport";
 
 export interface SynchronizationBuilder {
+    metadataIncludeExcludeRules: MetadataIncludeExcludeRules | undefined;
```

Comment:
Same as before ``MetadataIncludeExcludeRules`` instead of ``MetadataIncludeExcludeRules | undefined``

Reply:
Same as before also :)

Reply:
done!